docs/identities/get-started/setup.mdx

```diff
@@ -0,0 +1,190 @@
+import nextProdSetup from '!!raw-loader!./_common/code-examples/nextjs/lib/ory.prod.ts'
+import nextDevSetup from '!!raw-loader!./_common/code-examples/nextjs/lib/ory.local.ts'
+import goSetup from '!!raw-loader!./_common/code-examples/go/ory_client.go'
+import goProdSetup from '!!raw-loader!./_common/code-examples/go/ory_client_prod.go'
+import jsDevSetup from '!!raw-loader!./_common/code-examples/js/setupDev.js'
+import prodSetup from '!!raw-loader!./_common/code-examples/js/setup/setup.js'
+```
+
+This guide shows how to set up the necessary dependencies and configurations to integrate Ory's identity management features into
+your application.
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+1. An [Ory network account](https://console.ory.sh)
+2. Your Ory project id
+3. Your development environment set up with your framework of choice
+
+First, install the Ory SDK for your framework:
+
+```mdx-code-block
+<FrameworkCodeTabs>
+<TabItem value="javascript">
+```
+
+```shell
+npm install @ory/client-fetch --save
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="nextjs">
+```
+
+```shell
+npm install @ory/client-fetch --save
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="go">
+```
+
+```shell
+go get github.com/ory/client-go
+```
+
+```mdx-code-block
+</TabItem>
+</FrameworkCodeTabs>
+```
+
+## 2. Set up local development with Ory Tunnel
+
+For local development, you'll need to use Ory Tunnel to connect your local application with Ory's APIs:
+
+```mdx-code-block
+<Tabs
+  defaultValue="macos"
+  values={[
+    {label: 'macOS', value: 'macos'},
+    {label: 'Linux', value: 'linux'},
+  ]}>
+<TabItem value="macos">
+```
+
+```shell
+# Install Ory CLI using Homebrew
+brew install ory/tap/cli
+
+# Verify installation
+ory help
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="linux">
+```
+
+```mdx-code-block
+</TabItem>
+</Tabs>
+```
+
+After installing the CLI, start the tunnel to connect your local application with Ory's APIs:
+
+```shell
+# Start the tunnel (replace with your project id)
+ory tunnel --project <project-id> http://localhost:3000
+```
+
+:::tip
+
+To learn more about the Ory Tunnel, read the [dedicated section of the Ory CLI documentation](../../guides/cli/proxy-and-tunnel).
+
+:::
+
+## 3. Configure the SDK
+
+When using the tunnel, configure your SDK to use the local tunnel URL:
+
+```mdx-code-block
+<FrameworkCodeTabs>
+<TabItem value="javascript">
+```
+
+```mdx-code-block
+<CodeBlock language="js"> {jsDevSetup} </CodeBlock>
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="nextjs">
+```
+
+```mdx-code-block
+ <CodeBlock
+  language="ts"
+  title="lib/ory.ts"> {nextDevSetup} </CodeBlock>
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="go">
+```
+
+```mdx-code-block
+ <CodeBlock language="go" title="ory_client.go"> {goSetup} </CodeBlock>
+```
+
+```mdx-code-block
+</TabItem>
+</FrameworkCodeTabs>
+```
+
+For production environments, configure the SDK to use the Ory Network URL:
```

Comment:
The code for development and production will be shared - the developer will also only work in one code base. Can you please merge the code examples and just reduce the complexity?

Instead, explain what ORY_SDK_URL needs to be on local and on prod.